modules/extensions-support/src/test/java/org/mule/module/extension/internal/runtime/connector/petstore/PetStoreConnectorConfig.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.runtime.connector.petstore;
+
+import org.mule.extension.annotations.Configuration;
+import org.mule.extension.annotations.Parameter;
+import org.mule.extension.annotations.connector.Connector;
+
+@Configuration(name = "config")
+@Connector(PetStoreClientConnectionHandler.class)
```

Comment:
This is ok today, as this is the only parameter, but might there be other connector parameters in the future?